src/rpc/util.cpp

```diff
@@ -749,3 +750,64 @@ UniValue GetServicesNames(ServiceFlags services)
 
     return servicesNames;
 }
+
+void EnsureFileWritable(const std::string& filepath)
```

Comment:
I think the code could be simplified by passing in a `const fs::path&` here directly, because all call-sites start from that and use `.string()`.

Reply:
Ok, will adapt it.